src/main/java/staffconnect/logic/commands/AddMeetingCommand.java

```diff
@@ -92,10 +93,11 @@ private static Person addMeetingToPerson(Person personToEdit, Meeting meeting) {
         Set<Tag> currentTags = personToEdit.getTags();
         Set<Availability> currentAvailability = personToEdit.getAvailabilities();
         Set<Meeting> currentMeetings = new HashSet<>(personToEdit.getMeetings()); //to reduce coupling with Person
+        Favourite currentFavourite = personToEdit.getFavourite();
+
         Person editedPerson =
-            new Person(currentName, currentPhone, currentEmail, currentModule, currentFaculty, currentVenue,
-                       currentTags,
-                       currentAvailability);
+            new Person(currentName, currentPhone, currentEmail, currentModule, currentFaculty,
+                    currentVenue, currentTags, currentAvailability, currentFavourite);
```

Comment:
This line is outdated with my changes

Reply:
May I inquire what were your changes?

Reply:
I forgot to include in my comment, but I was referring to changes made as part of #130 . The above would be the following line as extracted from #130
```java

 Person editedPerson = PersonUtil.copyPerson(personToEdit);
````

Reply:
@Pluiexo I believe it's not updated to the changes you've made because your pull request has yet to be approved and merged into the master code.